.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,103 @@
+---
+name: Support Request
+about: Support request or question relating to KubeEdge
+labels: question
+
+---
+
+<!-- Please use this template while providing as much info as possible. Thanks!-->
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+<details><summary>Kubernets Version</summary>
+```console
```

Comment:
Looks like the ` ```console` doesn't work within the `<details>`? 
![image](https://user-images.githubusercontent.com/28776356/93476569-885e9a80-f92c-11ea-9e0b-461f99376412.png)

Reply:
Fix it, in fact it needs extra empty lines.